bin/bl-beepmein-pipemenu

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# pipeBeepmein
+# beepmein pipe menu for jgmenu
+
+# usage: pipeBeepmein | jgmenu --simple --icon-size=0
+
+# required
+command -v at >/dev/null 2>&1 || { >&2 echo "I need at: sudo apt install --no-install-recommends at" ; exit 1; }
+command -v gmrun >/dev/null 2>&1 || { >&2 echo "I need gmrun." ; exit 1; }
+
+# for display purposes
+future="$(date -d "+5 minutes" +%R)"
+
+# start to display jgmenu
+echo "Beepmein Alarm Clock (Time is now: $(date +%R)),x-terminal-emulator -T beepmein -e bash -c 'beepmein -h; echo "Press any key to close"; read -srn1' "
+#echo "Beepmein Alarm Clock (Time is now: $(date +%R))"
```

Comment:
If it's commented out, do we even need it in here?

Reply:
Will remove.